.github/workflows/rebase.yaml

```diff
@@ -0,0 +1,19 @@
+name: PR Rebase
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  rebase:
+    name: Rebase
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase')
+    if: github.event.comment.author_association == 'MEMBER' || github.event.comment.user.login == github.event.pull_request.user.login
```

Comment:
can i have multiple `if`s here or do i need to put it all in one?

Reply:
I think you can't, just because it's parsed as YAML. I like declarative YAML generally but these workflows do really push it to its limits for this kind of thing 😅

Reply:
doh 😞